onert-micro/onert-micro/include/OMConfig.h

```diff
@@ -17,13 +17,86 @@
 #ifndef ONERT_MICRO_CONFIG_H
 #define ONERT_MICRO_CONFIG_H
 
+#include <stdint.h>
+#include <stdlib.h>
+
 namespace onert_micro
 {
 
+/*
+ * OMTrainOptimizer - enum to store optimizers supported by training with onert-micro
+ */
+enum OMTrainOptimizer
+{
+  SGD,
+  ADAM,
+};
+
+/*
+ * OMMetrics - enum to store metrics supported by training with onert-micro
+ */
+enum OMMetrics
```

Comment:
@BalyshevArtem Note that nnfw api does not expose Metric Evaluation. That is, nnfw api will expose loss evaluation function.